### ISSUE TITLE
Couldn't decrypt the repo, please make sure you enter the right password!

### ISSUE THREAD

**Issue by edias** [fastlane/fastlane#4020]
### Issue Description

Match can't decrypt the repo and keep me asking the passphrase over and over again.

##### Complete output when running fastlane, including the stack trace and command used

```
[22:57:23]: Cloning remote git repo...
[22:57:28]: Migrating to new match...
[22:57:28]: Enter the passphrase that should be used to encrypt/decrypt your certificates
[22:57:28]: This passphrase is specific per repository and will be stored in your local keychain
[22:57:28]: Make sure to remember the password, as you'll need it when you run match on a different machine
Passphrase for Git Repo: ******
Type passphrase again: ******
[22:57:34]: 🔒  Successfully encrypted certificates repo
[22:57:34]: Cloning remote git repo...
[22:57:39]: Couldn't decrypt the repo, please make sure you enter the right password!
version: 256
class: "inet"
attributes:
    0x00000007 <blob>="match_git@bitbucket.org:username/myrepo.git"
    0x00000008 <blob>=<NULL>
    "acct"<blob>=<NULL>
    "atyp"<blob>="dflt"
    "cdat"<timedate>=0x32303136303430313130303034375A00  "20160401100047Z\000"
    "crtr"<uint32>=<NULL>
    "cusi"<sint32>=<NULL>
    "desc"<blob>=<NULL>
    "icmt"<blob>=<NULL>
    "invi"<sint32>=<NULL>
    "mdat"<timedate>=0x32303136303430313130303034375A00  "20160401100047Z\000"
    "nega"<sint32>=<NULL>
    "path"<blob>=<NULL>
    "port"<uint32>=0x00000000 
    "prot"<blob>=<NULL>
    "ptcl"<uint32>=0x00000000 
    "scrp"<sint32>=<NULL>
    "sdmn"<blob>=<NULL>
    "srvr"<blob>="match_git@bitbucket.org:username/myrepo.git"
    "type"<uint32>=<NULL>
password has been deleted.
[23:01:02]: Enter the passphrase that should be used to encrypt/decrypt your certificates
[23:01:02]: This passphrase is specific per repository and will be stored in your local keychain
[23:01:02]: Make sure to remember the password, as you'll need it when you run match on a different machine
Passphrase for Git Repo: 
```

### Environment

fastlane 1.70.0
match 0.4.0
rvm 1.26.11

**Comment by TKBurner** [comment]
Hi @edias! Thanks for reaching out. Do you have any special characters in your password? We are tracking an issue related to this right now.

Thanks!

**Comment by edias** [comment]
No, only characters and numbers.

**Comment by TKBurner** [comment]
Hi @edias. Can you send over your Match file? Thanks :+1:

**Comment by edias** [comment]
I'm just running match development on the terminal

**Comment by TKBurner** [comment]
Hi @edias. Did you run Match init? You should have a Matchfile in your fastlane dir.

Thanks!

**Comment by edias** [comment]
git_url "git@bitbucket.org:edias/myproject_provisioningprofiles.git"

type "development" # The default type, can be: appstore, adhoc or development

# app_identifier "tools.fastlane.app" 
# username "user@fastlane.tools" # Your Apple Developer Portal username

# For all available options run `match --help`
# Remove the # in the beginning of the line to enable the other options

**Comment by TKBurner** [comment]
@edias Thanks for getting back to me. We're going to look into this over here. Thanks for working with me! :+1:

**Comment by edias** [comment]
You are welcome, do you guys have any ETA for this issue?

**Comment by TKBurner** [comment]
Thanks for following up @edias! I'm going to flag this as help wanted to see if we can get some community attention. I'm following up internally too :+1:

**Comment by yakubbaev** [comment]
I'm having the same issue now

**Comment by yakubbaev** [comment]
It appears that I messed up my passphrase. Everything is ok for me now. Thanks for your great work!

**Comment by GantMan** [comment]
I'm seeing this issue.  Seeing @yakubbaev messed up his passphrase,  I figured I will just delete the certs and re-download them, but it appears match is forever trying to ask me to decrypt files I now deleted.   How do I start over?

**Comment by TKBurner** [comment]
Hi @GantMan! Hope you are doing well. Can you send over the full output? Thanks!

**Comment by GantMan** [comment]
This occurred because I was storing my files in the same repo as the project.

Somehow, moving this to a separate repo entirely fixed this.   I'm not sure if this is related, but I'm going to suggest this in the future.  

It seems standard to use a separate repo, but in some cases I can see that not being a need.  It would be nice to store certs/provisions in the same repo as the app.

**Comment by TKBurner** [comment]
@GantMan Thanks for following up. Right now match is designed to work with it's own repo. In the instruction we say to use a NEW private repo. So glad to hear you got up and running! Thanks for working with us and have a great weekend 👍

**Comment by gekitz** [comment]
I'm having a similar problem and I can't figure out what the problem is, I attached a screenshot with my output. Help is very much appreciated :D 

I'm not sure what the output means, did match successful decrypt my certificate and fails afterwards or what exactly is the problem? (I'm wondering bc of the green "Decrypted" output)

https://www.dropbox.com/s/gwnwkfsqksaiitb/Screenshot%202016-05-23%2015.53.29.png?dl=0

**Comment by gekitz** [comment]
Was of course my fault, forgot to encrypt my prov.profiles 💃

**Comment by TKBurner** [comment]
@gekitz No problem! Really glad to hear you got up and running. Hope that you have a wonderful week 👍

**Comment by Ankita942016** [comment]
Hello,
I am facing the same issue

Couldn't decrypt the repo, please make sure you enter the right password!
keychain: "/Users/anksoni/Library/Keychains/Bot"
version: 256
class: "inet"
attributes:
    0x00000007 <blob>="match_https://usistudio66@bitbucket.org/usistudio66/provisional-profile.git"
    0x00000008 <blob>=<NULL>
    "acct"<blob>=<NULL>
    "atyp"<blob>="dflt"
    "cdat"<timedate>=0x32303136303533313130303732335A00  "20160531100723Z\000"
    "crtr"<uint32>=<NULL>
    "cusi"<sint32>=<NULL>
    "desc"<blob>=<NULL>
    "icmt"<blob>=<NULL>
    "invi"<sint32>=<NULL>
    "mdat"<timedate>=0x32303136303533313130303732335A00  "20160531100723Z\000"
    "nega"<sint32>=<NULL>
    "path"<blob>=<NULL>
    "port"<uint32>=0x00000000 
    "prot"<blob>=<NULL>
    "ptcl"<uint32>=0x00000000 
    "scrp"<sint32>=<NULL>
    "sdmn"<blob>=<NULL>
    "srvr"<blob>="match_https://usistudio66@bitbucket.org/usistudio66/provisional-profile.git"
    "type"<uint32>=<NULL>
password has been deleted.


What could be the issue?

**Comment by TKBurner** [comment]
@Ankita942016 Thanks for writing in. Can you let me know if you have your match repo completely empty except the match files? You can't store your project code there. Thanks!

**Comment by Ankita942016** [comment]
Yes it has no code just provisional profile and certificates.

**Comment by TKBurner** [comment]
@Ankita942016 Thanks for following up. Did you run the setup on a new private repo or a previously used one? Thanks!

**Comment by Ankita942016** [comment]
A new repo

**Comment by pkoder** [comment]
I was seeing the same error as well. 
My problem was I never encrypted the certificates and provisioning profile and added them straight
into my repo.
When running "match appstore" it would try to clone the repo and immediately prompt me for a password.
Obviously this password did not exist.
I then tried "match change_password" in the hopes of initiating that step but it would check my keychain
for an existing match password that also did not exist.

What ended up working was cleaning out the repo and running "match appstore"

This then created a new certificate and new provisioning profile, downloaded, installed, encrypted etc... and everything worked as expected.

The problem is I just now duplicated a certificate and provisioning profile.
Is there a method in match where I can seed my git repo with existing certs and profiles?
